usr/src/uts/armv8/os/startup.c

```diff
@@ -1226,6 +1228,12 @@ startup_modules(void)
 	 */
 	seg_init();
 
+	/*
+	 * Gather CPU information
+	 */
+	if (cpuinfo_init() != 0)
```

Comment:
I think I have the same question about whether we can panic before this happens?

Reply:
https://gist.github.com/r1mikey/fdcea906a2bcad446ca7acef63672779

This one is safe, and since PSCI is up it all works as expected.